src/components/sdk/VideoTileGrid/index.tsx

```diff
@@ -38,11 +38,14 @@ interface Props extends BaseProps {
   noRemoteVideoView?: React.ReactNode;
   /** The layout of the grid. */
   layout?: Layout;
+  /** add local user name in localtile */
+  localuserName?:string;
```

Comment:
The nameplate can be "Me" or any name builders want to set for the local video tile namplate, so you are right it can be username but it can also be other name mapping username to some name. Would it be better to instead name this prop as `localVideoNameplate` or `localUserNameplate`?

Reply:
"localVideoNameplate" seems fine will go with that